tests/run_tests.sh

```diff
@@ -25,16 +25,28 @@ if [ ${python_version[0]} -lt 3 ] || [[ "${python_version[0]}" -eq 3 && "${pytho
     exit 1
 fi
 
+function check_pep440_compatible_version() {
+    local version="${1}"
+    ${python_bin} -c "from setuptools._vendor.packaging.version import Version; Version(\"${version}\")"
+    return $?
+}
+
 tmp_dir_path="${SCRIPT_DIR}/${TMP_DIR_NAME}"
 
 invalid_package=0
 failed_test=0
 
 package_list=`find ${PACKAGES_PATH} -name "*.zip"`
 for pkg in ${package_list}; do
-    if [[ ${pkg} =~ (fake_bpy_modules_([a-z0-9\.]+)-[0-9]{8}).zip ]]; then
+    if [[ ${pkg} =~ (fake_bpy_modules_([a-z0-9\.]+)-(.*)).zip ]]; then
         pkg_dir_name=${BASH_REMATCH[1]}
         blender_version=${BASH_REMATCH[2]}
+        pkg_version=${BASH_REMATCH[3]}
+        if ! check_pep440_compatible_version "${pkg_version}"; then
+            echo "'${pkg_dir_name}' version '${pkg_version}' does not conform with PEP440. Ignoring."
```

Comment:
I think this check does not ignore the package because of `            ((invalid_package+=1))` and checking on L67.
So, it is good for user to remove the message `Ignoring.`.

Reply:
@nutti can you elaborate? I don't think I understand what you mean.

Reply:
Only I want to do is deleting `Ignore.` word.
Here is a suggestion code.

```suggestion
            echo "'${pkg_dir_name}' version '${pkg_version}' does not conform with PEP440."
```

Reply:
Oh I understand. Do you want to write "Invalid package: " at the beginning of the output? That's makes it probably more clear.

Reply:
@grische 

Yes, nice suggestion!
I think it's more clear.

Reply:
Changed